### PR TITLE
Adding Ruby 2.7.z and Rails 6.y.z releases to the CircleCI build configuration

### LINKED ISSUE #93: Add support for Ruby 2.7.z releases
Ruby 2.7.0 was released on 12/25/2019, and in accordance with the charter of the current phase of the Component Maintenance Working Group, the CircleCI configuration for this should be updated.

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #93 and #94

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,27 +41,51 @@ jobs:
...
+          rails_version: 5.2.4
       - bundle_and_test:
           name: ruby2-5_rails5-1
-          ruby_version: 2.5.5
+          ruby_version: 2.5.7
           rails_version: 5.1.7
+      - bundle_and_test:
+          name: ruby2-4_rails6-0
+          ruby_version: 2.4.9
+          rails_version: 6.0.2
```
Comment (cjcolvar) [inline .circleci/config.yml]: I though rails 6 requires ruby 2.5+

**Comment by jrgriffiniii** [comment]
Review this, this appears to be related to a sprockets dependency issue.

**Comment by jrgriffiniii** [comment]
This is finally resolved, I just need to squash the commits for this and open this for review.

**Comment by jrgriffiniii** [comment]
This is now ready for review.

**Pull request merged**